rasterio/gdal.pxi

```diff
@@ -698,6 +698,54 @@ cdef extern from "gdal_alg.h" nogil:
             void * pTransformArg, double * padfGeoTransformOut, int * pnPixels,
             int * pnLines, double * padfExtent, int nOptions)
 
+cdef extern from "gdal_utils.h" nogil:
+    ctypedef struct GDALBuildVRTOptions:
+        bint bStrict
+        char *pszResolution
+        int bSeparate
+        int bAllowProjectionDifference
+        double we_res
+        double ns_res
+        int bTargetAlignedPixels
+        double xmin
+        double ymin
+        double xmax
+        double ymax
+        int bAddAlpha
+        int bHideNoData
+        int nSubdataset
+        char* pszSrcNoData
+        char* pszVRTNoData
+        char* pszOutputSRS
+        int *panSelectedBandList
+        int nBandCount
+        char* pszResampling
+        char** papszOpenOptions
+        bint bUseSrcMaskBand
+        # allow or suppress progress monitor and other non-error output
+        int bQuiet
+        # the progress function to use
+        GDALProgressFunc pfnProgress
+        # pointer to the progress data variable
+        void *pProgressData
+
+    ctypedef struct GDALBuildVRTOptionsForBinary:
```

Comment:
same here. This is an opaque non-public structure

Reply:
Thanks @rouault :+1: